srcpkgs/vice/files/desktop-install.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh -e
+dir=$DESTDIR/usr/share/applications/
+[ "$1" = install ] && mkdir -p $dir && shift && for f; do cp $f $dir; done
```

Comment:
Is there a reason why you didn't just add xdg-utils to hostmakedepends?

Reply:
@paper42 it didn't install properly into DESTDIR, I'm not sure if there's some XDG_ env var that needs to be set for it to work

Reply:
Did you try XDG_DATA_DIRS?

Reply:
Ah, I did but it needs directories to exist already